projects/plugins/jetpack/tests/e2e/bin/env.sh

```diff
@@ -39,6 +42,8 @@ if [ "${1}" == "start" ]; then
 	start_env
 elif [ "${1}" == "reset" ]; then
 	reset_env
+elif [ "${1}" == "gb-setup" ]; then
+	gb_setup ${2}
```

Comment:
Why do we need to pass the 2nd arg here?

Reply:
I mixed up the files. I've removed it.